validator/pandora/service_test.go

```diff
@@ -0,0 +1,68 @@
+package pandora
+
+import (
+	"context"
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+	logTest "github.com/sirupsen/logrus/hooks/test"
+	"strings"
+	"testing"
+	"time"
+)
+
+// MockPandoraService method returns a mocked pandora service using
+// mocked rpc client and mocked rpc server.
+func MockPandoraService(endpoint string, dialPandoraFn DialRPCFn) (*Service, error) {
+	mockedPandoraService, err := NewService(context.Background(), endpoint, dialPandoraFn)
+	if err != nil {
+		return nil, err
+	}
+	return mockedPandoraService, nil
+}
+
+// TestStart_OK method checks that service starts successfully or not
+func TestStart_OK(t *testing.T) {
+	hook := logTest.NewGlobal()
+	pandoraService, err := MockPandoraService(HttpEndpoint, DialInProcRPCClient)
+	require.NoError(t, err, "Error in preparing pandora mock service")
+
+	pandoraService.Start()
```

Comment:
are you sure that this runs through test at all? (I have mentioned go func behaviour in previous file). To check if this works and you'll prove me wrong you can insert anything dummy into one of the `wants` and it should fail

Reply:
I'll check also for myself, maybe I'm wrong

Reply:
I was wrong. Explained above

Reply:
Thanks :)